main.py

```diff
@@ -400,6 +400,8 @@ def ui_access_wrapper(self, *args, **kwargs):
     get_redirect_route(
         r'/createhandler/autosave_draft/<exploration_id>',
         editor.EditorAutosaveHandler),
+    get_redirect_route(
+        r'/gettopunresolvedanswers', editor.TopUnresolvedAnswersHandler),
```

Comment:
/createhandler/get_top_unresolved_answers (maybe with <exploration_id> at the end), I think? For consistency with the ones above.

Reply:
Will do

Reply:
Modified.